src/components/NavBar/PostButtonModal.tsx

```diff
@@ -4,6 +4,8 @@ import CloseIcon from "@mui/icons-material/Close";
 const styles = {
   dialog: {
     maxHeight: "90%",
+    scroll: "paper",
+    ".MuiDialog-scrollPaper": { display: "flex", alignItems: "flex-start" },
```

Comment:
Do you need `display: "flex"`? I think that is already applied as default from the MUI component

Reply:
Yes, it is the default. I removed it.